applications/welcome/models/tables.py

```diff
@@ -17,48 +17,3 @@
 
 ## after defining tables, uncomment below to enable auditing
 # auth.enable_record_versioning(db)
-
-import datetime
-
-## These correspond to client tables.
-
-db.define_table('logs',
-                Field('device_id'),
-                Field('timestamp', 'datetime', default=datetime.datetime.utcnow()),
-                Field('module'),
-                Field('level', 'integer'), #  int, 0 = most important.
-                Field('message', 'text'),
```

Comment:
These are in the welcome app. Not in client. These were added to welcome through some other app or probably removed during their merge with master.